_config.php

```diff
@@ -19,7 +19,6 @@
 
 ShortcodeParser::get('default')
 	->register('file_link', array('File', 'handle_shortcode'))
-	->register('embed', array('Oembed', 'handle_shortcode'))
```

Comment:
We'll need to have a shortcode handler that handles the bridge to the 3rd party module.